src/host/main.cpp

```diff
@@ -61,16 +63,8 @@ bool asynchost::TCPImpl::alloc_quota_logged = false;
 size_t asynchost::UDPImpl::remaining_read_quota =
   asynchost::UDPImpl::max_read_quota;
 
-std::chrono::microseconds asynchost::TimeBoundLogger::default_max_time(10'000);
-
-void print_version(size_t)
-{
-  std::cout << "CCF host: " << ccf::ccf_version << std::endl;
-  std::cout << "Platform: "
-            << nlohmann::json(ccf::pal::platform).get<std::string>()
-            << std::endl;
-  exit(0);
-}
+std::chrono::microseconds asynchost::TimeBoundLogger::default_max_time(
+  10'000); // NOLINT(cppcoreguidelines-avoid-magic-numbers)
```

Comment:
Would prefer to minimise the `NOLINT`s - can we do `#define DEFAULT_TIME_BOUND_LOGGER_MAX_TIME_MICROSECONDS 10'000` or similar?